### PR TITLE
Tweaks to eslint configs, and gets all 4 packages running in CI

### PR DESCRIPTION
**Pull request by quaelin**

### CODE REVIEW

File: packages/eslint-plugin-change/rules/prefer-object-spread-to-lodash.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
 const isLodashCall = node => node.callee && node.callee.type === 'MemberExpression' && node.callee.object.name === '_';
-const isExtendOrAssignInCall = node =>
-  isLodashCall(node) && (node.callee.property.name === 'extend' || node.callee.property.name === 'assignIn');
+// eslint-disable-next-line prettier/prettier
```
Comment (tobobo) [inline packages/eslint-plugin-change/rules/prefer-object-spread-to-lodash.js]: does prettier fight with eslint on the correct formatting of this line? If so, that might indicate that the config isn't quite right...

**Comment by tobobo** [comment]
Are the lint changes in change-codemod and eslint-plugin-change indicative of the fact that the configuration wasn't quite right earlier? Or is it not quite right now? The fact that prettier needs to be disabled for a line makes me wonder if it isn't the latter...

File: packages/eslint-plugin-change/rules/prefer-object-spread-to-lodash.js (hunk 1 above)
Comment (quaelin) [reply packages/eslint-plugin-change/rules/prefer-object-spread-to-lodash.js]: Yeah, it does...

**Comment by quaelin** [comment]
Ok, I think it's working now.  The order in which configs are extended matters, so we need to make sure that we extend `change-base` _first_, and then `prettier` _after_ that.  Since we can't extend `change-base` from the `eslint-config-change-base` code itself, anything we put in the top level .eslintrc was causing problems for it, so I got rid of that top level config file and just put explicit configs in each module directory.

**Comment by tobobo** [comment]
LGTM with merge conflicts fixed

**Pull request merged**